<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="picture">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>24</y>
      <width>741</width>
      <height>711</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="x1spinBox">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>50</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="y1spinBox">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>50</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="x2spinBox">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>50</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="y2spinBox">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>50</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="x1Label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="y1Label">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="x2Label">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>20</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="y2Label">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>20</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="drawButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>390</y>
      <width>421</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Рисовать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compareButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>620</y>
      <width>421</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Сравнить визуальные хар-ки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cleanButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>660</y>
      <width>421</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>110</y>
      <width>457</width>
      <height>218</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="standart">
       <property name="text">
        <string>QPainter.drawLine</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="cda">
       <property name="text">
        <string>ЦДА</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="brezan">
       <property name="text">
        <string>Брезенхем</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="brezan_int">
       <property name="text">
        <string>Брезенхем (целые)</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="brezan_step">
       <property name="text">
        <string>Брезенхем (с устр. ступенчататости)</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="chooseColorButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>350</y>
      <width>171</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать цвет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bgColorButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>350</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет фона</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="xcSpinBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>470</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="x1Label_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>440</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>XC</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="ycSpinbBox">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>470</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="y1Label_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>440</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>YC</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="lenSpinBox">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>470</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angleSpinBox">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>470</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
   <widget class="QLabel" name="y1Label_3">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>440</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Длина</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="y1Label_4">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>440</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Угол</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>350</y>
      <width>121</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Черный</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1333</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
